src/debugger/android/androidPlatform.ts

```diff
@@ -19,17 +19,21 @@ import {Package} from "../../common/node/package";
 export class AndroidPlatform implements IAppPlatform {
     private extensionMessageSender: ExtensionMessageSender;
 
+    private static MULTIPLE_DEVICES_ERROR = "error: more than one device/emulator";
```

Comment:
As discussed offline, because of the success patterns, we need to do this on the .catch method, not on the .then